burn-wgpu/src/kernel/reduce/tune/prod_dim.rs

```diff
@@ -0,0 +1,116 @@
+use burn_compute::tune::{AutotuneOperation, AutotuneOperationSet};
```

Comment:
there is large copy-pasta from sum implementation

what is the recommendation here? parametrize the core implementation and call them with sum / prod arguments?

Reply:
I guess we'll want to refactor this so that all reduce operations share some core. The list of reduction operations is growing (there's also this PR #1136 coming up with more autotuned reduce) so we can't afford to have this many duplicates. For now you can keep it as is but I'll open an issue for refactoring this.